### PR TITLE
IGNITE-16772 BigInteger can't be serialized by User Object Serialization

### PR DESCRIPTION
**Pull request by rpuch**
https://issues.apache.org/jira/browse/IGNITE-16772

### CODE REVIEW

**Review by ptupitsyn — DISMISSED** [review]
@rpuch please update `ItThinClientComputeTest` - look for `// TODO IGNITE-16772` there.

**Comment by rpuch** [comment]
@ptupitsyn thanks for noting, I've uncommented that line, the test seems to pass

File: modules/network/src/test/java/org/apache/ignite/internal/network/serialization/marshal/DefaultUserObjectMarshallerWithArbitraryObjectsTest.java [hunk 1]
```diff
@@ -553,6 +554,13 @@ void nonLatin1InStringFieldIsSupported() throws Exception {
         assertThat(unmarshalled.value, is("щ"));
     }
 
+    @Test
+    void supportsBigInteger() throws Exception {
```
Comment (ibessonov) [inline modules/network/src/test/java/org/apache/ignite/internal/network/serialization/marshal/DefaultUserObjectMarshallerWithArbitraryObjectsTest.java:558]: Every other test here looks like "xxxIsSupported()", maybe we should rename it

**Review by ibessonov — COMMENTED** [review]
I don't think that I understand this code well enough, you should really ask @SammyVimes for the review

File: modules/network/src/test/java/org/apache/ignite/internal/network/serialization/marshal/DefaultUserObjectMarshallerWithArbitraryObjectsTest.java (hunk 1 above)
Comment (rpuch) [reply modules/network/src/test/java/org/apache/ignite/internal/network/serialization/marshal/DefaultUserObjectMarshallerWithArbitraryObjectsTest.java]: Renamed the method

**Comment by rpuch** [comment]
Thanks, already did

**Review by SammyVimes — APPROVED** [review]
LGTM

**Comment by SammyVimes** [comment]
Merged to the main branch

**Comment by rpuch** [comment]
Thanks for the review!

**Pull request closed without merging**